OpenRA.Mods.Common/Traits/BotModules/HarvesterBotModule.cs

```diff
@@ -58,11 +57,10 @@ protected override void TraitEnabled(Actor self)
 			domainIndex = world.WorldActor.Trait<DomainIndex>();
 			resLayer = world.WorldActor.TraitOrDefault<ResourceLayer>();
 			claimLayer = world.WorldActor.TraitOrDefault<ResourceClaimLayer>();
-			botOrderManager = self.Owner.PlayerActor.Trait<BotOrderManager>();
 			scanForIdleHarvestersTicks = Info.ScanForIdleHarvestersInterval;
 		}
 
-		void ITick.Tick(Actor self)
+		void IBotTick.BotTick(IBot bot)
 		{
 			if (IsTraitDisabled)
```

Comment:
This isn't strictly necessary, as `HackyAI.Tick()` will check it itself already before calling modules.